examples/channel-hello-world/TARGETS

```diff
@@ -0,0 +1,28 @@
+cpp_binary(
+  name = 'tcp-server',
+  headers = subdir_glob([('.', '**/*.h')]),
+  srcs = [
+    'HelloStreamRequestHandler.cpp',
+    'HelloStreamSubscription.cpp',
+    'StreamHelloWorld_Server.cpp',
```

Comment:
Did you mean to copy the TARGETS file over?  If so then we should at least do `s|Stream|Channel|g` on them.